package.json

```diff
@@ -4,7 +4,9 @@
   "description": "1. clone repo 2. install dependencies 3. run script 4. clean app.ts",
   "main": "index.js",
   "scripts": {
-      "blitz": "cp template/app.template.ts src/app.ts && ./node_modules/ts-node/dist/bin.js src/app.ts"
+    "blitz": "cp templates/app.template.ts src/app.ts && ./node_modules/ts-node/dist/bin.js src/app.ts",
```

Comment:
maybe we can remove this but call sandbox blitz instead?

Reply:
Yes, we can. It nodemon was actually supposed to be before the ts-node part. That's also the reason that I created #5. Let's just forget that and use this sandbox script instead :D